pkg/resources/manager/manager.go

```diff
@@ -437,6 +475,16 @@ func (r *resourceManager) getMachineClassAvailibilityAsString() string {
 	return removeSeparatorFromEnd(result)
 }
 
+func (r *resourceManager) getAvailableVMSlotsAsString() string {
+	count := r.availableVMSlots
+
+	if count < 0 {
```

Comment:
cannot be this little confusing? availablevmslots can be negative what i saw and this value is function is mainly use for debugging.

Reply:
It can be negative if there are existing machines in store and the libvirt-provider is initialized with a lower VMLimit. However, in such cases, detailed information is available during the initialization of the resource-manager, which provides clarity to the backend team.

The `availableVMSlots` variable indicates the number of VMs that can still be created on the host. Therefore, if the limit is already reached, it should be set to 0.